src/safeposix/syscalls/fs_calls.rs

```diff
@@ -814,6 +823,48 @@ impl Cage {
         ret
     }
 
+    pub fn sbrk(&self, brk: u32) -> i32 {
+        let mut vmmap = self.vmmap.write();
+        let heap = vmmap.find_page(0).unwrap().clone();
```

Comment:
instead of 0 lets define this as a constant for readability, also add comment why